app/models/observation.rb

```diff
@@ -1,5 +1,5 @@
 class Observation < ApplicationRecord
   belongs_to :encounter
   belongs_to :user
-  belongs_to :observable, polymorphic: true
+  belongs_to :observable, polymorphic: true, optional: true
```

Comment:
Is this change related to merging soft-deleted records?

Reply:
Thanks for catching this, removed!